frontend/public/actions/features.ts

```diff
@@ -18,10 +18,11 @@ import store from '../redux';
 import { GroupModel, UserModel, VolumeSnapshotContentModel } from '../models';
 import { ClusterVersionKind } from '../module/k8s';
 import { receivedResources } from './k8s';
-import { setClusterID, setCreateProjectMessage, setUser } from './common';
+import { setClusterID, setCreateProjectMessage } from './common';
 import client, { fetchURL } from '../graphql/client';
 import { SSARQuery } from './features.gql';
 import { SSARQueryType, SSARQueryVariables } from '../../@types/console/generated/graphql-schema';
+import { setUser } from '@console/dynamic-plugin-sdk/src/app/core/actions/core';
```

Comment:
This seems oddly specific... are you avoiding some circle reference issue? Not not just the SDK package?

Reply:
make sense, it was an auto import. I doubt there would be a circular reference issue. will update